apps/web/src/app/app/(dashboard)/[workspaceId]/monitors/page.tsx

```diff
@@ -23,7 +23,7 @@ export default async function MonitorPage({
       </Header>
       {monitors.map((monitor, index) => (
         <Container key={index} title={monitor.name} description={monitor.url}>
-          <ActionButton {...{ ...monitor, workspaceId }} />
+          <ActionButton props={monitor} />
```

Comment:
Will be probably be then:

```suggestion
          <ActionButton {...monitor} />
```